src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -43,44 +43,45 @@ export default class AuthCodeRedirectHandler implements IRedirectHandler {
     return !!(url.query && url.query.code && url.query.state);
   }
 
-  async handle(redirectUrl: string): Promise<ISolidSession> {
-    if (!(await this.canHandle(redirectUrl))) {
-      throw new ConfigurationError(`Cannot handle redirect url ${redirectUrl}`);
-    }
-    const url = new URL(redirectUrl, true);
-    const localUserId = url.query.state as string;
-    const [codeVerifier, redirectUri] = await Promise.all([
-      (await this.storageUtility.getForUser(
-        localUserId,
-        "codeVerifier",
-        true
-      )) as string,
-      (await this.storageUtility.getForUser(
-        localUserId,
-        "redirectUri",
-        true
-      )) as string
-    ]);
+  async handle(redirectUrl: string): Promise<ISessionInfo> {
+    throw new Error("Not Implemented");
+    // if (!(await this.canHandle(redirectUrl))) {
+    //   throw new ConfigurationError(`Cannot handle redirect url ${redirectUrl}`);
+    // }
+    // const url = new URL(redirectUrl, true);
+    // const localUserId = url.query.state as string;
+    // const [codeVerifier, redirectUri] = await Promise.all([
+    //   (await this.storageUtility.getForUser(
+    //     localUserId,
+    //     "codeVerifier",
+    //     true
+    //   )) as string,
+    //   (await this.storageUtility.getForUser(
+    //     localUserId,
+    //     "redirectUri",
+    //     true
+    //   )) as string
+    // ]);
 
-    /* eslint-disable @typescript-eslint/camelcase */
-    await this.tokenRequester.request(localUserId, {
-      grant_type: "authorization_code",
-      code_verifier: codeVerifier as string,
-      code: url.query.code as string,
-      redirect_uri: redirectUri as string
-    });
-    /* eslint-enable @typescript-eslint/camelcase */
+    // /* eslint-disable @typescript-eslint/camelcase */
+    // await this.tokenRequester.request(localUserId, {
+    //   grant_type: "authorization_code",
+    //   code_verifier: codeVerifier as string,
+    //   code: url.query.code as string,
+    //   redirect_uri: redirectUri as string
+    // });
+    // /* eslint-enable @typescript-eslint/camelcase */
 
-    url.set("query", {});
+    // url.set("query", {});
 
-    const session = await this.sessionCreator.getSession(localUserId);
-    if (!session) {
-      throw new Error("There was a problem creating a session.");
-    }
-    session.neededAction = this.redirector.redirect(url.toString(), {
-      redirectByReplacingState: true
-    });
+    // const session = await this.sessionCreator.getSession(localUserId);
```

Comment:
Ditto

Reply:
Fixed in 505dc8c6b6872a881ab84ea3642b04ff9d260feb